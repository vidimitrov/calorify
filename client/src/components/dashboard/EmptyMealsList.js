import React from 'react';
import styled from 'styled-components';
import img from '../../assets/img/empty.png';

const Wrapper = styled.div`
  text-align: center;
`;

const Image = styled.img`
  width: 150px;
`;

const Headline = styled.h2`
  color: #f6cd48;
`;

const EmptyMealsList = () => (
  <Wrapper>
    <Image src={img} />
    <Headline>Add your first meal</Headline>
  </Wrapper>
);

export default EmptyMealsList;
