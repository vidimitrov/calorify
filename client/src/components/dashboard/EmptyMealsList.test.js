import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import EmptyMealsList from './EmptyMealsList';

it('Dashboard/EmptyMealsList component renders correctly', () => {
  const tree = renderer.create(<EmptyMealsList />).toJSON();
  expect(tree).toMatchSnapshot();
});
