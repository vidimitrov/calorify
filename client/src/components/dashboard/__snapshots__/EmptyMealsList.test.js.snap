// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard/EmptyMealsList component renders correctly 1`] = `
.c0 {
  text-align: center;
}

.c1 {
  width: 150px;
}

.c2 {
  color: #f6cd48;
}

<div
  className="c0"
>
  <img
    className="c1"
    src="empty.png"
  />
  <h2
    className="c2"
  >
    Add your first meal
  </h2>
</div>
`;
